shapes/plot-description/surface-strew-lithology/shapes.ttl

```diff
@@ -0,0 +1,334 @@
+
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX ns1: <urn:property:>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX reg: <http://purl.org/linked-data/registry#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX sosa: <http://www.w3.org/ns/sosa/>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<urn:shapes:plot-description:surface-strew-lithology:feature-type>
+    a
+        sh:PropertyShape ,
+        <urn:class:Requirement> ;
+    dcterms:source "TERN Ecosystem Surveillance Ecological Monitoring Protocols" ;
+    reg:status reg:statusSubmitted ;
+    sh:description "TERN's ecologists have determined the feature type is _landform_, defined by the link:https://www.publish.csiro.au/book/5230/[Australian Soil and Land Survey Field Handbook]." ;
+    sh:hasValue <http://linked.data.gov.au/def/tern-cv/2cf3ed29-440e-4a50-9bbc-5aab30df9fcd> ;
```

Comment:
This should be *land surface* with the value `http://linked.data.gov.au/def/tern-cv/8282fb22-4135-415c-8ca2-317860d102fb`. 

See https://linkeddata.tern.org.au/information-models/tern-ontology/dev-guide/dawe-protocol/plot-description-module/surface-strew-lithology.

Can you also fix it for `outcrop-lithology` as well please? See https://linkeddata.tern.org.au/information-models/tern-ontology/dev-guide/dawe-protocol/plot-description-module/outcrop-lithology